### PR TITLE
fern meta tags

### PR DESCRIPTION
**Pull request by itelo**
[fern docs](https://buildwithfern.com/docs/spec/extensions)

### CODE REVIEW

**Review by codetheweb — COMMENTED** [review]
could you add a comment with the link to the relevant docs for these properties?

**Review by codetheweb — COMMENTED** [review]
sorry, don't see any doc url yet

File: packages/nextlove/src/generators/generate-openapi/fern-sdk-utils.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  if (lastPart.startsWith("[") && lastPart.endsWith("]")) {
+    // Convert [param] to by_param
+    const param = lastPart.slice(1, -1)
+    return `by_${param}`
+  }
+
+  return lastPart
+}
+
+function transformPathToFernSdkGroupName(path: string) {
```
Comment (itelo) [inline packages/nextlove/src/generators/generate-openapi/fern-sdk-utils.ts:15]:
```suggestion
// fern docs: https://buildwithfern.com/docs/spec/extensions
function transformPathToFernSdkGroupName(path: string) {
```

**Comment by itelo** [comment]
my bad, I thought you wanted the doc link in the PR description

**Review by codetheweb — APPROVED** [review]

**Pull request merged**